fcm/atmosphere/_mars_atm_api.py

```diff
@@ -102,7 +120,7 @@ def _load_and_parse_txt_file(url, timeout=2):
     assert atmosphere.shape[1] == 1,\
         "expected only two columns, got {:d}".format(atmosphere.shape[1] + 1)
     atmosphere.columns = ["Density (kg/m3)"]
-    atmosphere.index.name = "altitude above MOLA_0 (m)"
+    atmosphere.index.name = "altitude"
```

Comment:
Looks to me like this line was redundant before, and still is, as it's overwritten on line 70 anyway. => should be removed

Reply:
Yup, done.